spec/spec_helper.rb

```diff
@@ -16,7 +16,8 @@
   SimpleCov.start do
     track_files 'lib/**/*.rb'
 
-    add_filter 'lib/puppet/resource_api/version.rb'
+    add_filter 'lib/ace/version.rb'
+    add_filter 'lib/ace.rb'
```

Comment:
this file does not exist, and `lib/ace.rb` above can be deleted if it's empty. 

I'd rather not have those landmines lying about here.